operations/helm/charts/grafana-agent/templates/containers/_agent.yaml

```diff
@@ -25,6 +25,11 @@
   ports:
     - containerPort: {{ .Values.agent.listenPort }}
       name: http-metrics
+    {{- range $portMap := .Values.agent.extraPorts }}
+    - containerPort: {{ $portMap.port }}
```

Comment:
Should this be targetPort instead of port? (since port refers to the service port and targetPort refers to the port number from the container)

Reply:
Totally right! I'll change this now.